pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -206,10 +206,31 @@ public static bool ReadFromFile(string fileName, out string loggedSkylineDocumen
             {
                 using (var alert = new AlertDlg())
                 {
-                    alert.Message = string.Format(
-                        AuditLogStrings.AuditLogList_ReadFromFile_An_exception_occured_while_reading_the_audit_log,
-                        fileName);
-                    alert.DetailMessage = ex.StackTrace;
+                    Exception exception = ex;
+                    StringBuilder msgBuilder = new StringBuilder("");
+                    List<string> stackList = new List<string>();
+
+
```

Comment:
Would our existing GetStackTraceText method work here?

Reply:
Well, I did not know about it, of course. It would have done, but it uses non-localized strings and it looks a bit inconsistent to me. Why would only part of the text visible to the user be translated.
I'll keep my code for now.